README.md

```diff
@@ -67,14 +67,22 @@ AVS_KEY=$(cast wallet new --json | jq -r '.[0].private_key')
 # This will show the address, so you can confirm it was properly added when listing operators
 cast wallet addr --private-key "$AVS_KEY"
 
-docker run --rm --network host --env-file .env -v ./.nodes:/root/.nodes  --entrypoint /wavs/register.sh wavs-middleware "$AVS_KEY"
+export WAVS_SERVICE_MANAGER_ADDRESS=$(jq -r '.addresses.WavsServiceManager' .nodes/avs_deploy.json)
```

Comment:
Great stuff. Pull out the variables we actually need from the json so we can provide this to the operators.

Ideally, we would have a separate OPERATORS.md or such in a project that contained this, and includes the actual service manager address. But this is great for testing